modules/FvwmButtons/parse.c

```diff
@@ -515,7 +519,38 @@ static void ParsePanel(
 			n = 0;
 			(*panel_flags).panel_indicator = 1;
 			*indicator_size = 0;
-			sscanf(s, "%d%n", indicator_size, &n);
+			sscanf(s, "%d %s%n", indicator_size, indicator_in_out, &n);
```

Comment:
We might want to change this string to put size limits?  `"%d %s<NUM>%n"` -- currently `%s` is unbounded, so could crash with large input.